custom/import_xml.php

```diff
@@ -190,24 +193,31 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<title><?php xl('Import Patient Demographics','e'); ?></title>
+    <?php Header::setupHeader(); ?>
+    <title><?php echo xlt('Import Patient Demographics XML'); ?></title>
 </head>
 <body class="body_top" onload="javascript:document.forms[0].form_import_data.focus()">
-
-<p><?php xl('Paste the data to import into the text area below:','e'); ?></p>
-
-<center>
-<form method='post' action="import_xml.php">
-
-<textarea name='form_import_data' rows='10' cols='50' style='width:95%'></textarea>
-
-<p>
-<input type='submit' name='form_import' value=<?php xl('Import Patient','e','\'','\''); ?> /> &nbsp;
-<input type='button' value=<?php xl('Cancel','e','\'','\''); ?> onclick='window.close()' /></p>
+<form method='post' action="import_xml.php" onsubmit="return top.restoreSession()">
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="form-group"></div>
+                <div class="form-group">
```

Comment:
if don't do this, then the below textarea hugs right at the top of the form and it looks really weird. LEt me know if there's a better way to bring in spacing and I'll change this (in both the export and import scripts).